README.md

```diff
@@ -716,11 +702,43 @@ _NOTE: Skip these steps if you are deploying without promoting changes._
    and resolve conflicts in necessary. Conflict resolution should only be
    necessary if cherry-picks occured on the target branch.
 
-4. The merge may have affected `README.md`, the file you are looking at right
+3. The merge may have affected `README.md`, the file you are looking at right
    now. Reopen the file now to ensure you are following the updated version.
 
+4. Now you need to create the release notes. (Skip this step if noA link to the release
+   notes document can be found either in the #dcp-ops channel on HCA Slack or in
+   the Google Drive folder mentioned in the [DCP release SOP].
+
+   To produce the list of changes for the DCP release notes, run
+
+   ```
+   git log --pretty=oneline --topo-order --abbrev-commit
+   ```
+
```

Comment:
```suggestion
   Recently, we started to merge backported hotfixes rather than cherry picking. 
   I found that topological order wasn't insuffcient. I found that 
   `--graph --date-order` works better than `--topo-order` in those cases.
```

Reply:
Or sth along those lines.

Reply:
replaced with
`git LAST_RELEASE_TAG..HEAD --format="%C(auto) %h %s" --no-merges`